src/main/java/net/openhft/chronicle/network/cluster/ClusterContext.java

```diff
@@ -226,7 +226,10 @@ public void cluster(Cluster<T, C> cluster) {
         this.cluster = cluster;
     }
 
-    protected abstract void defaults();
+    protected void defaults() {
```

Comment:
I wonder if that breaks existing bytecode

Reply:
It does not. I have tried running QE `mvn surefire:test` before and after replacing the jar and all good

Reply:
We will probably need to add calls to `super.defaults()` in the places `defaults` was previously implemented, now overridden? I know there's two tests that do it and potentially more in other repos.

Reply:
DG, Map-Enterprise and Queue-Enterprise override it and all are now working as expected - see https://github.com/ChronicleEnterprise/Chronicle-Queue-Enterprise/pull/391 & https://github.com/ChronicleEnterprise/Chronicle-Map-Enterprise/pull/112

All the tests are green - [build all branch](https://teamcity.chronicle.software/project/Chronicle_BuildAll?branch=feature%2Fimprove-pauser-defaults&mode=builds) - so I think we are good